client/signup/steps/website-content/wordpress-media-upload.tsx

```diff
@@ -88,6 +108,7 @@ export interface MediaUploadData {
 	URL?: string;
 	uploadID?: string;
 	mediaIndex: number;
+	thumbnailUrl?: string;
```

Comment:
Minor: The interface seems to be very similar to the `Media` type defined in the schema file. Any chance we can replace this interface with the `Media` type?

Reply:
Yes this is a good refactor to do . I will pick this up as a followup task.